README.md

```diff
@@ -2,11 +2,59 @@
 
 ## Installation
 ```sh
+$ docker-compose up --build
+```
+You should configure `.env` file to ensure that your instance works as intended.
+
+Example `.env` config:
+```sh
+DJANGO_SETTINGS_MODULE=healthcheck.settings.dev
+SECRET_KEY=secret-key
+DATABASE_URL=psql://user:password@db:5432/table
+ALLOWED_HOSTS=*
+TURN_API_KEY=turn-api-key
+CELERY_BROKER_URL=redis://redis:6379/0
+CELERY_RESULT_BACKEND=redis://redis:6379/0
+CELERY_ACCEPT_CONTENT=application/json
+CELERY_TASK_SERIALIZER=json
+CELERY_RESULT_SERIALIZER=json
+```
+`PostgreSQL` is used by default in docker container named `db`. It's best to not change that. You can change the following enviromental variables in `docker-compose.yml`:
+```yml
+db:
+    environment: 
+        - POSTGRES_PASSWORD=django
+        - POSTGRES_USER=django
+        - POSTGRES_DB=healthcheck
+```
+You also need to include those in `DATABASE_URL`.
+You should re-build your local container after adding new dependencies.
+
+## Running
+```sh
 $ docker-compose up
 ```
 
-## Running tests
+You can access shell by either installing all dependencies from `setup.py`, `requirements.txt` and `requirements-dev.txt` in your `venv` and running
+```sh
+$ python manage.py shell
+```
+ or by connecting to docker container shell using following command:
+```sh
+$ docker-compose run django bash
 ```
+
+## Submitting a PR
+Before submitting a PR make sure you have ran tests
+```sh
 $ pip install -e .
 $ python manage.py test
 ```
+and formatted your code. To do so, run:
+```
+$ isort **/*.py
```

Comment:
We could also add these to precommit hook.

Reply:
We aren't using precommit in this repository but i would consier adding one